src/images/sprites/mnoe-sprites.less

```diff
@@ -1,45 +1,66 @@
+/*
+Icon classes can be used entirely standalone. They are named after their original file names.
 
-.mnoe-icon {
-  background-image: url('../images/sprites/mnoe-sprites.png');
+Example usage in HTML:
+
+`display: block` sprite:
+<div class="icon-home"></div>
+
+To change `display` (e.g. `display: inline-block;`), we suggest using a common CSS class:
+
+// CSS
+.icon {
+  display: inline-block;
 }
 
+// HTML
+<i class="icon icon-home"></i>
+*/
 .mnoe-icon-cc-icons-mno-cc-amex {
-  background-position: -0px -0px;
+  background-image: url(../images/sprites/mnoe-sprites.png);
+  background-position: -175px -32px;
   width: 46px;
   height: 30px;
 }
 .mnoe-icon-cc-icons-mno-cc-jcb {
-  background-position: -0px -30px;
+  background-image: url(../images/sprites/mnoe-sprites.png);
+  background-position: -221px -32px;
   width: 46px;
   height: 30px;
 }
+.mnoe-icon-cc-icons-mno-cc-lock {
+  background-image: url(../images/sprites/mnoe-sprites.png);
+  background-position: -140px -32px;
+  width: 35px;
+  height: 46px;
+}
 .mnoe-icon-cc-icons-mno-cc-mastercard {
-  background-position: -0px -60px;
+  background-image: url(../images/sprites/mnoe-sprites.png);
+  background-position: -140px -78px;
   width: 46px;
   height: 30px;
 }
 .mnoe-icon-cc-icons-mno-cc-visa {
-  background-position: -0px -90px;
+  background-image: url(../images/sprites/mnoe-sprites.png);
```

Comment:
Asking for an Angular learning opportunity. Can you also access assets through `@/images/sprites/mnoe-sprites.png`?

Reply:
I thought that was only in vue?

Reply:
It's TS aliases defined with the `paths` property in the `compilerOptions` of the `tsconfig.json` file.
eg:
```json
"compilerOptions": {
  "baseUrl": "./",
      "paths": {
        "@models/*": [
          "src/app/_models/*"
        ],
     ...
  }
}
```